tests/transaction_test.cpp

```diff
@@ -28,24 +28,6 @@ auto g_blk_samples = Lazy([] { return samples::createMockDagBlkSamples(0, NUM_BL
 
 struct TransactionTest : BaseTest {};
 
-TEST_F(TransactionTest, double_verification) {
```

Comment:
this should not be removed, but modified to we know that it is working for `period_data`